include/library/spdm_responder_conformance_test_lib.h

```diff
@@ -57,14 +57,14 @@ void spdm_responder_conformance_test (void *spdm_context,
 #define   SPDM_RESPONDER_TEST_CASE_CHALLENGE_AUTH_VERSION_MISMATCH     4
 #define   SPDM_RESPONDER_TEST_CASE_CHALLENGE_AUTH_UNEXPECTED_REQUEST   5
 #define   SPDM_RESPONDER_TEST_CASE_CHALLENGE_AUTH_INVALID_REQUEST      6
-#define   SPDM_RESPONDER_TEST_CASE_CHALLENGE_AUTH_SUCCESS_12_A1B1C1    11
-#define   SPDM_RESPONDER_TEST_CASE_CHALLENGE_AUTH_SUCCESS_12_A1B2C1    12
-#define   SPDM_RESPONDER_TEST_CASE_CHALLENGE_AUTH_SUCCESS_12_A1B3C1    13
-#define   SPDM_RESPONDER_TEST_CASE_CHALLENGE_AUTH_SUCCESS_12_A1B4C1    14
-#define   SPDM_RESPONDER_TEST_CASE_CHALLENGE_AUTH_SUCCESS_12_A2B1C1    15
-#define   SPDM_RESPONDER_TEST_CASE_CHALLENGE_AUTH_SUCCESS_12_A2B2C1    16
-#define   SPDM_RESPONDER_TEST_CASE_CHALLENGE_AUTH_SUCCESS_12_A2B3C1    17
-#define   SPDM_RESPONDER_TEST_CASE_CHALLENGE_AUTH_SUCCESS_12_A2B4C1    18
+#define   SPDM_RESPONDER_TEST_CASE_CHALLENGE_AUTH_SUCCESS_12_13_A1B1C1    11
+#define   SPDM_RESPONDER_TEST_CASE_CHALLENGE_AUTH_SUCCESS_12_13_A1B2C1    12
+#define   SPDM_RESPONDER_TEST_CASE_CHALLENGE_AUTH_SUCCESS_12_13_A1B3C1    13
+#define   SPDM_RESPONDER_TEST_CASE_CHALLENGE_AUTH_SUCCESS_12_13_A1B4C1    14
+#define   SPDM_RESPONDER_TEST_CASE_CHALLENGE_AUTH_SUCCESS_12_13_A2B1C1    15
+#define   SPDM_RESPONDER_TEST_CASE_CHALLENGE_AUTH_SUCCESS_12_13_A2B2C1    16
+#define   SPDM_RESPONDER_TEST_CASE_CHALLENGE_AUTH_SUCCESS_12_13_A2B3C1    17
+#define   SPDM_RESPONDER_TEST_CASE_CHALLENGE_AUTH_SUCCESS_12_13_A2B4C1    18
```

Comment:
I think we might want to discuss how to manage the new version.  (e.g. 1.4 in the future)

Option1: Change the name to `12_13`, and change to `12_13_14` again.
Option2: Do not change the name.
Option3: Change the name - use `12_ABOVE`
Option4: Add `13` separately.